roles/nova-common/defaults/main.yml

```diff
@@ -27,7 +27,7 @@ nova:
   scheduler_host_subset_size: 1
   libvirt_bin_version: 1.3.1-1ubuntu10.8~cloud0
   python_libvirt_version: 1.3.1-1ubuntu1~cloud0
-  qemu_kvm_version: 2.0.0+dfsg-2ubuntu1.30
+  qemu_kvm_version: 1:2.5+dfsg-5ubuntu10.9~cloud0
```

Comment:
seems like qemu-kvm version in ubuntu mirror got bumped to 2.0.0+dfsg-2ubuntu1.33. Your changes looks fine. Waiting on QA team regression results before merging this.